.gitignore

```diff
@@ -17,3 +17,6 @@ gen
 del
 static/prod.*
 Config/client_session_key.aes
+
+# Ignore IHP lib.
```

Comment:
I'd rename this as well as the `IHP lib` has a different meaning in IHP itself (build/ihp-lib, see https://github.com/digitallyinduced/ihp/blob/master/IHP/LibDir.hs)

```suggestion
# Ignore locally checked out IHP version
```